src/AccordionContext.tsx

```diff
@@ -0,0 +1,3 @@
+import React from 'react';
+
+export default React.createContext<string | null>(null);
```

Comment:
nit: _technically_ this file doesn't have any JSX in it, so I'm not sure if it should have the `.tsx` extension.